server/app/views/v1/grid_secrets/index.json.jbuilder

```diff
@@ -3,4 +3,8 @@ json.secrets @grid_secrets do |grid_secret|
   json.name grid_secret.name
   json.created_at grid_secret.created_at
   json.updated_at grid_secret.updated_at
+  json.services grid_secret.services do |grid_service|
```

Comment:
This smells like it will kill `kontena secret list` performance again (#2234), with a large number of secrets, because it's O(N) `grid_services` queries? Also hard to fix with `includes`... do these have to be included in the `index` view, or is it enough to have them in in the show view?

Reply:
Good point. Related services would be nice to see without revealing the secret value. Any workarounds?

Reply:
The docs on mongoid `includes` say:

>  Note: This will work for embedded relations that reference another collection via belongs_to as well.

But the `GridServiceSecret` is just `field :name, type: String`... if that embedded document was changed to a `belongs_to :grid_secret` relation on the `has_many :grid_service_secrets`, then would `@grid.grid_secrets.includes(:grid_service_secrets)` work?